packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -118,12 +121,6 @@ export const BaseButtonStyles = (
             display: flex;
             pointer-events: none;
         }
-        .start {
```

Comment:
Replaced with `gap`, which is closer to the intent and safer than needing to style `start` and `end` independently everywhere.